### ISSUE TITLE
Build for CentOS images fails

### ISSUE THREAD

**Issue by jokay** [zabbix/zabbix-docker#835]
##### SUMMARY

The build for the CentOS based images, e.g. `zabbix/zabbix-server-pgsql:5.0.11-centos` fails, see [here](https://github.com/zabbix/zabbix-docker/runs/2437681424).

In general, wouldn't it make sense to create the Release [5.0.11](https://github.com/zabbix/zabbix-docker/releases/tag/5.0.11) only after every image did successfully build?

**Comment by dotneft** [comment]
It is not a good idea, because it looks like CentOS base image is broken now...

**Comment by jokay** [comment]
Seems indeed to affect all CentOS based images.

**Comment by dotneft** [comment]
Most of them, they have missed package... only one and only on CentOS 8. The base image is not updated for 4 months ((

**Comment by jokay** [comment]
We would use the `-alpine` image if there were `curl` available *(needed for some custom notification scripts)*.

But we want to use the original image, not another custom image by adding `curl` to the original `-alpine` image :wink:

**Comment by jokay** [comment]
We use a `curl` command inside a bash script to send SMS which is setup as **media  type**.

![sms-notif](https://user-images.githubusercontent.com/18613935/116147193-b5073b80-a6df-11eb-8f23-6b859dbabae7.png)

Not sure if this can be done per user using webhooks?

**Comment by dotneft** [comment]
SMS is some HTTP API (Rest API)?

**Comment by jokay** [comment]
Yes

**Comment by dotneft** [comment]
So I think you can use webhooks :-) Please check official documentation and try to move CURL request to JavaScript curl request: https://www.zabbix.com/documentation/current/manual/config/notifications/media/webhook/webhook_examples

**Comment by dotneft** [comment]
Build fixed!

**Comment by jokay** [comment]
Thx a lot :+1:

**Comment by jokay** [comment]
@dotneft it builds but is [not yet available](https://hub.docker.com/r/zabbix/zabbix-server-pgsql/tags?page=1&ordering=last_updated&name=5.0.11-) on hub.docker.com.

**Comment by jokay** [comment]
> @dotneft it builds but is not yet available on [hub.docker.com](https://hub.docker.com/r/zabbix/zabbix-server-pgsql/tags?page=1&ordering=last_updated&name=5.0.11-).

Created a follow-up issue https://github.com/zabbix/zabbix-docker/issues/836.

**Comment by jokay** [comment]
> So I think you can use webhooks :-) Please check official documentation and try to move CURL request to JavaScript curl request: https://www.zabbix.com/documentation/current/manual/config/notifications/media/webhook/webhook_examples

@dotneft this works indeed using webhooks for [Gotify](https://gotify.net/) as well :+1:

![gotify-webhook](https://user-images.githubusercontent.com/18613935/117567940-b3198100-b0be-11eb-9c81-3d3df5f692ad.png)

```js
// Gotify webhook
try {
    var rs,
        payload,
        params = JSON.parse(value),
        endpoint = params.URL + '/message?token=' + params.To,
        rq = new CurlHttpRequest();

    if (typeof params.HTTPProxy === 'string' &&
        params.HTTPProxy.trim() !== '') {
        rq.setProxy(params.HTTPProxy);
    }

    var data = {
        title: params.Subject,
        message: params.Message,
        priority: 5
    }

    payload = JSON.stringify(data);
    Zabbix.Log(4, '[ Gotify webhook ] Sending rq: ' + payload);

    rq.AddHeader('Content-Type: application/json');
    rs = rq.Post(endpoint, payload);

    Zabbix.Log(4, '[ Gotify webhook ] Received rs with status code ' +
        rq.Status() + '\n' + rs
    );

    if (rq.Status() !== 200) {
        throw 'Request failed with status code ' + rq.Status() +
        '. Check debug log for more information.';
    }

    return 'OK';
}
catch (ex) {
    Zabbix.Log(4, '[ Gotify webhook ] ERROR: ' + ex);

    throw 'Sending failed: ' + ex;
}
```

**Comment by jokay** [comment]
I wonder why [template literals (ES6+)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) are not supported, it would make the whole script more readable?

**Comment by dotneft** [comment]
Excellent! Thank you for the information!
